misc/package/Data Files/MWSE/core/meta/class/aliases.lua

```diff
@@ -0,0 +1,3305 @@
+-- This file is autogenerated. Do not edit this file manually. Your changes will be ignored.
```

Comment:
We generally split most things into their own files. Is there a reason we went for a single file this time instead of splitting them up into actionFlag.lua and similar this time?